addons/website/static/src/snippets/s_share/000.js

```diff
@@ -20,6 +20,11 @@ const ShareWidget = publicWidget.Widget.extend({
                 return href.replace(urlRegex, function (match, a, b, c) {
                     return a + url + c;
                 }).replace(titleRegex, function (match, a, b, c) {
+                    if (href.indexOf('https://wa.me/?text') !== -1) {
+                        // For WhatsApp, everything needs to be sent via 'text' parameter. Page URL will
+                        // be parsed and converted into hyperlink automatically by the platform
```

Comment:
Can you give an example of the output?
Something like: 
```js
// WhatsApp does not support the "url" GET parameter.
// Instead we need to include the url within the passed "text" parameter, merging everything together.
// e.g of output: 
// https://wa.me/?text=%20Conference%20%7C%20My%20Website%20https%3A%2F%2Fmycompany.odoo.com%2Fevent%2Fconference-2021-06-26-3%2Fregister
// see https://faq.whatsapp.com/general/chats/how-to-use-click-to-chat/ for more details
```

Also, are we sure that there will be a space character between the "title" and the "url"? I"m guessing this could mess the output.